modules/ROOT/pages/to-find-info.adoc

```diff
@@ -4,336 +4,120 @@ include::_attributes.adoc[]
-
-A search value matches an asset value if either of these is true:
+Categories are specific groupings of assets set by the administrator of a business group or an organization owner.
+* Tag
++
+Keyword or term that describes a specific search. 
+* Lifecycle state
++
+Lifecycle state can be stable, deprecated, or development.
 
-* The search value is entirely lowercase.
-* The letter case of the search value and the asset value match.
+Filter options appear when they are available within an organization. 
 
-Examples:
+To filter search results:
 
-[%header%autowidth.spread]
-|===
-|Search |Category |Matches?
-|category:my-key = my-value |my-key:my-value |Yes
-|category:my-key = MY-VALUE |my-key:MY-VALUE |Yes
-|category:my-key = my-value |my-key:MY-VALUE |Yes
-|category:"my key" = "my value" |my key:my value |Yes
-|category:"key" = "value" |my key:my value |No
-|category:key = value |my-key:my-value |No
-|category:this.* = value |this.is.my.key:value |Yes
-|category:*my.key = value |this.is.my.key:value |Yes
-|category:this.* = value |this.is.my.key:some-value |No
-|category:this.* = some-* |this.is.my.key:some-value |No
-|===
+. From the Exchange landing page, select a filter option.
+. Click an option from the menu and click *Apply*.
+. To clear a filter option, click *Clear filter*.
+. To clear all filter options, click *Clear all filters*.
 
-You can search for a combination of different categories. You cannot search for multiple values of the same category.
+== Sort
 
-Multiple search terms use an implicit `AND`. `OR` is not supported.
+Assets can be sorted by the following options:
 
-These search terms return all assets marked as both system APIs and Salesforce products:
+* Relevance
++
+Sorts the matching results by where a match is found in the asset metadata, such as name, title, pages, and API specification metadata. API specification metadata includes endpoints and types.
+* Name
++
+Sorts by asset name in ascending alphabetical order.
+* Last modified
++
+Sorts the results by the most recently modified assets at the top, for example, the latest published assets or assets that have new versions recently added. This is the default for all business groups except assets that are provided by Mulesoft.
+* Rating
++
+Sorts by descending order by the review rating of an asset (stars on an asset card). The highest rated assets is at the top. This is the default order for the Provided by MuleSoft and Global Exchange.
+* Featured
++
+Sorts by placing a featured selection of assets at the top of the results. Other assets that are not featured assets are ordered by last modified.
 
-`category:"API Type" = "System API" category:"Product" = "Salesforce"`
+The sorting options appear when they are available within an organization.
 
-=== Search by Custom Fields
+To sort the ranking of search results, use the *Sort by* drop-down menu to select an option: Relevance (default), Name, Last modified, Rating. Relevance, or Featured.
```

Comment:
```suggestion
To sort the ranking of search results, use the *Sort by* drop-down menu to select an option: Relevance, Name, Last modified, Rating, or Featured.
```

I would take the "(default)" remark out, since the default options depends on many factors explained above.